docs/caching/includes/azure-redis-app-host.md

```diff
@@ -21,20 +19,147 @@ dotnet add package Aspire.Hosting.Azure.Redis
+<!-- markdownlint-enable MD033 -->
+
+The preceding Bicep is a module that provisions an Azure Cache for Redis with the following defaults:
+
+- `location`: The location of the Azure Cache for Redis resource. The default is the location of the resource group.
+- `principalId`: The principal ID of the Azure Cache for Redis resource.
+- `principalName`: The principal name of the Azure Cache for Redis resource.
+- `sku`: The SKU of the Azure Cache for Redis resource. The default is `Basic` with a capacity of `1`.
+- `enableNonSslPort`: The non-SSL port of the Azure Cache for Redis resource. The default is `false`.
+- `disableAccessKeyAuthentication`: The access key authentication of the Azure Cache for Redis resource. The default is `true`.
+- `minimumTlsVersion`: The minimum TLS version of the Azure Cache for Redis resource. The default is `1.2`.
+- `redisConfiguration`: The Redis configuration of the Azure Cache for Redis resource. The default is `aad-enabled` set to `true`.
+- `tags`: The tags of the Azure Cache for Redis resource. The default is `aspire-resource-name` set to the name of the Aspire resource, in this case `redis`.
+- `redis_contributor`: The contributor of the Azure Cache for Redis resource, with an access policy name of `Data Contributor`.
+- `connectionString`: The connection string of the Azure Cache for Redis resource.
+
+In addition to the Azure Cache for Redis, it also provisions an Azure Firewall rule to allow all Azure IP addresses. Finally, an administrator is created for the Redis server, and the connection string is outputted as an output variable. The generated Bicep is a starting point and can be customized to meet your specific requirements.
+
+#### Customize provisioning infrastructure
+
+All .NET Aspire Azure resources are subclasses of the <xref:Aspire.Hosting.Azure.AzureProvisioningResource> type. This type enables the customization of the generated Bicep by providing a fluent API to configure the Azure resources—using the <xref:Aspire.Hosting.AzureProvisioningResourceExtensions.ConfigureInfrastructure``1(Aspire.Hosting.ApplicationModel.IResourceBuilder{``0},System.Action{Aspire.Hosting.Azure.AzureResourceInfrastructure})> API. For example, you can configure the `kind`, `consistencyPolicy`, `locations`, and more. The following example demonstrates how to customize the Azure Cache for Redis resource:
+
+:::code language="csharp" source="../../snippets/azure/AppHost/Program.ConfigureRedisInfra.cs" id="configure":::
+
+The preceding code:
+
+- Chains a call to the <xref:Aspire.Hosting.AzureProvisioningResourceExtensions.ConfigureInfrastructure*> API:
+  - The `infra` parameter is an instance of the <xref:Aspire.Hosting.Azure.AzureResourceInfrastructure> type.
+  - The provisionable resources are retrieved by calling the <xref:Azure.Provisioning.Infrastructure.GetProvisionableResources> method.
+  - The single <xref:Azure.Provisioning.Redis.RedisResource> is retrieved.
+  - The `RedisVersion` is set to `"6.0"`.
```

Comment:
The default is `latest`, which is currently `6.0`. Hard-coding this into our docs doesn't seem right. Maybe just showing how to adjust the sku/capacity is enough.

Reply:
I removed it.